lib/v2/luogu/daily.js

```diff
@@ -21,7 +21,7 @@ module.exports = async (ctx) => {
             link,
             author: firstPost.find('p').eq(1).text(),
             guid: `${link}#${issueHeading}`,
-            pubDate: parseDate(issueHeading.match(/(\d{4} 年 \d{2} 月 \d{2} 日)/)[1], 'YYYY 年 MM 月 DD 日'),
+            pubDate: parseDate(issueHeading.match(/(\d{4} 年 \d{2} 月 \d{1,2} 日)/)[1], 'YYYY 年 M 月 D 日'),
```

Comment:
```suggestion
            pubDate: parseDate(issueHeading.match(/(\d{4} 年 \d{1,2} 月 \d{1,2} 日)/)[1], 'YYYY 年 M 月 D 日'),
```
You may need to change the regex as well